README.md

```diff
@@ -0,0 +1,62 @@
+### **API response prototype**
+```json
+{
+  "links": {
+    "self": "http://microblog.com/posts?page[number]=0&page[size]=2",
+    "next": "http://microblog.com/posts?page[number]=2&page[size]=5"
```

Comment:
Looks way better now, but I'm not sure if square brackets in URL is valid

Reply:
[Json Api specification](https://jsonapi.org/format/#query-parameters-families) gives such an example of querry parameters:

> /?page[offset]=0&page[limit]=10

On the other hand [RFC3986 ](https://datatracker.ietf.org/doc/html/rfc3986#section-2.2) treats square brackets as 'reserved characters':

> These characters are called
   "reserved" because they may (or may not) be defined as delimiters by
   the generic syntax, by each scheme-specific syntax, or by the
   implementation-specific syntax of a URI's dereferencing algorithm.